assets/Script/technology/techGame1.js

```diff
@@ -0,0 +1,75 @@
+const { changeScene } = require('../util/sceneUtils');
+const { MENU_SCENE } = require('../constants');
+const MAX_BLOCK_NUMBER = 6;
+
+const generateNumberBlocks = () => {
+    return Math.floor(Math.random() * MAX_BLOCK_NUMBER);
+};
+
+const setupEventHandlers = (that) => {
+  that.homeSprite.on('mousedown', () => {
+    changeScene(MENU_SCENE);
+  });
+};
+
+cc.Class({
+    extends: cc.Component,
+
+    properties: {
+        speed : 200
+    },
+
+    onLoad () {
+        this.homeSprite = this.node.getChildByName('home_button');
+        this.blockSprite = this.node.getChildByName('block')
+
+        var numberBlocks = generateNumberBlocks();

```

Comment:
Can we please use `const` and `let` rather than `var`?
Give this a read: https://medium.com/javascript-scene/javascript-es6-var-let-or-const-ba58b8dcde75
Basically: `const > let > var`.

Reply:
will fix